src/LaddaButton.jsx

```diff
@@ -74,6 +74,8 @@ class LaddaButton extends Component {
     if (props.loading !== this.props.loading) {
       if (props.loading) {
         this.laddaInstance.start()
+      } else if (props.disabled) {
+        this.laddaInstance.disable()
```

Comment:
For the first point: nop, it's not necessary. [`Disable()` already calls `stop()`](https://github.com/hakimel/Ladda/blob/0218e92c78bda78931292905cfc332d0b01fe13c/js/ladda.js#L185). Otherwise the unit tests would have failed.

For the second point: I think it's pretty straightforward without comment, but as you want. 😉